docker-compose.yml

```diff
@@ -6,17 +6,27 @@ services:
       context: .
       dockerfile: Dockerfile
     command: ["gunicorn", "-b", "0.0.0.0:5000", "-w", "5", "conbench:application", "--access-logfile=-", "--error-logfile=-", "--preload"]
+    volumes:
+      - ${PWD}/_conbench-coverage-dir:/etc/conbench-coverage-dir
     environment:
       APPLICATION_NAME: "Conbench"
       DB_USERNAME: "postgres"
+      # From the docker-compose docs: "By default Compose sets up a single
+      # network for your app. Each container for a service joins the default
+      # network and is both reachable by other containers on that network, and
+      # discoverable by them at a hostname identical to the container name."
+      # That is why the DNS name `db` works.
       DB_HOST: "db"
       DB_PASSWORD: "postgres"
       DB_NAME: "postgres"
       DB_PORT: "5432"
       FLASK_APP: "conbench"
       FLASK_ENV: "development"
+      FLASK_DEBUG: "true"
       REGISTRATION_KEY: "code"
       SECRET_KEY: "Person, woman, man, camera, TV"
+      # Default log level is INFO, change to DEBUG as needed.
+      # CONBENCH_LOG_LEVEL_STDERR: DEBUG
```

Comment:
CC @austin3dickey

Reply:
Great! This would be nice to add to the README. Right now, it seems the README isn't recommending the best way to run tests locally:

https://github.com/conbench/conbench/blob/c086edf17a451002d278fe10b837c5a06edd0db8/README.md?plain=1#L150-L152

It would be cool to add something like `-e CONBENCH_LOG_LEVEL_STDERR=DEBUG` to this, and probably the `down` and `build` commands too.

Reply:
> it seems the README isn't recommending the best way to run tests locally:

Yes, let's change that! https://github.com/conbench/conbench/pull/468